### PR TITLE
Ft user can update profile

### PR DESCRIPTION
**Pull request by precious-ijege**
User can update Profile

### CODE REVIEW

File: flight_backend/api/serializers/users.py [hunk 1]
```diff
@@ -39,6 +39,18 @@ def create(self, validated_data):
         user.save()
         return user
 
+    def update(self, instance, validated_data):
```
Comment (egichuri) [inline flight_backend/api/serializers/users.py:42]: any reason we're overriding the default update method?

Comment (precious-ijege) [reply flight_backend/api/serializers/users.py:42]: I think you have to provide logic for the update method. I am not sure it works if you dont define it.

Comment (precious-ijege) [reply flight_backend/api/serializers/users.py:42]: you can check this out https://www.django-rest-framework.org/api-guide/serializers/

Comment (egichuri) [reply flight_backend/api/serializers/users.py:42]: sorry.. just saw you're using `APIView` on for the view. will you need to allow users to be added using means other than registration?

Comment (precious-ijege) [reply flight_backend/api/serializers/users.py:42]: No, Users will be added by registration alone.

**Review by egichuri — APPROVED** [review]

**Pull request merged**